Bottom: 2e5ed82c06743dbc09d48578c5c87648a8352e6d
Top:    cedd911a38c057e2c2111b9d4ccf4114e1b42f1e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 18:03:49 +0000

Revert "hid/gtk/ghid_layer_selector.c: Use g_malloc for the new_layer structure"

This reverts commit 38ac0fe9b2b4f758cbc2e67488c041706a034559.


---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 2b04063..432e57e 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -643,7 +643,7 @@ ghid_layer_selector_add_layer (GHidLayerSelector *ls,
       free_ldata (ls, new_layer);
     }
 
-  new_layer = g_malloc (sizeof (*new_layer));
+  new_layer = malloc (sizeof (*new_layer));
 
   gtk_list_store_set (ls->list_store,
                       &iter,
